docs/book/src/topics/crd-changes/v1alpha4-to-v1alpha5.md

```diff
@@ -0,0 +1,84 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [v1alpha4 compared to v1alpha5](#v1alpha4-compared-to-v1alpha5)
+  - [Conversion from v1alpha4 to v1alpha5](#conversion-from-v1alpha4-to-v1alpha5)
+  - [API Changes](#api-changes)
+    - [`OpenStackCluster`](#openstackcluster)
+      - [Managed API LoadBalancer](#managed-api-loadbalancer)
+      - [Major Changes to Ports and Networks](#major-changes-to-ports-and-networks)
+    - [`OpenStackMachine`](#openstackmachine)
+      - [Rename of `status.error{Reason,Message}` to `status.failure{Reason,Message}`](#rename-of-statuserrorreasonmessage-to-statusfailurereasonmessage)
+      - [Changes to `rootVolume`](#changes-to-rootvolume)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# v1alpha4 compared to v1alpha5
+
+## Conversion from v1alpha4 to v1alpha5
+
+CAPO is able to automatically convert your old resources into the new API Version.
```

Comment:
I think we can move this to `docs/book/src/topics/crd-changes/index.md`. Further a short notice that customers must change their resources (i.e., yaml files, source code, ...) as well.

Reply:
Done